scripts/codemods/typed_flags.py

```diff
@@ -0,0 +1,191 @@
+# SPDX-License-Identifier: MIT
+
+import importlib
+import textwrap
+import types
+from typing import List, Optional
+
+import libcst as cst
+import libcst.codemod.visitors as codevisitors
+import libcst.matchers as m
+from libcst import codemod
+
+from disnake import flags
+
+BASE_FLAG_CLASSES = (flags.BaseFlags, flags.ListBaseFlags)
+
+MODULES = ("disnake.flags",)
```

Comment:
might be unused?

Reply:
oops

Reply:
https://github.com/DisnakeDev/disnake/pull/778/commits/085a509e26967ef7e920e4347f1abc0a840f9f5d

Reply:
https://github.com/DisnakeDev/disnake/pull/778/commits/085a509e26967ef7e920e4347f1abc0a840f9f5d